FlashDevelop/MainForm.cs

```diff
@@ -1700,6 +1701,9 @@ protected override bool ProcessCmdKey(ref Message msg, Keys keyData)
              */
             else if (ShortcutKeysManager.IsValidExtendedShortcutFirst(keyData) && !appSettings.DisableExtendedShortcutKeys)
             {
+                if (ShortcutManager.AllShortcuts.FirstOrDefault(s => s.First == keyData) == ShortcutKeys.None)
+                    return false;
+
```

Comment:
When I wrote that, I didn't realise that was a fix for the mnemonics.
While it does work, I wouldn't really want to change the behaviour like this.
If it's urgent, I don't mind merging this, but I will definitely be changing this later.